nilearn/plotting/surf_plotting.py

```
@@ -422,6 +426,248 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
+        values are surface mesh geometries as accepted by plot_surf_stat_map.
+        If None, plot_img_on_surf will use Freesurfer's 'fsaverage5'.
+
+    mask_img : Niimg-like object or None, optional (default=None)
+        The mask is passed to vol_to_surf.
+        Samples falling out of this mask or out of the image are ignored
+        during projection of the volume to the surface.
+        If ``None``, don't apply any mask.
+
+    inflate : bool, optional (default=False)
+        If True, display images in inflated brain.
+        If False, display images in pial surface.
+
+    views : list, optional (default=['lateral', 'medial'])
+        A list containing all views to display.
+        The montage will contain as many rows as views specified by
+        display mode. Order is preserved, and left and right hemispheres
+        are shown on the left and right sides of the figure.
+
+    hemispheres : list, optional (default=['left', 'right'])
+        Hemispheres to display
+
+    output_file : str, optional (default=None)
+        The name of an image file to export plot to. Valid extensions
+        are: *.png*, *.pdf*, *.svg*. If output_file is not None,
+        the plot is saved to a file, and the display is closed. Return
+        value is None.
+
+    title : str, optional (default=None)
+        Place a title on the upper center of the figure.
+
+    colorbar : bool, optional (default=True)
+        If *True*, a symmetric colorbar of the statistical map is displayed.
+
+    vmax : float, optional (default=None)
+        Upper bound for plotting of stat_map values.
+
+    threshold : float, optional (default=None)
+        If None is given, the image is not thresholded.
+        If a number is given, it is used to threshold the image,
+        values below the threshold (in absolute value) are plotted
+        as transparent.
+
+    symmetric_cbar : bool or 'auto', optional, default 'auto'
```

symmetric_cbar is silently ignored (as discussed during the sprint simply removing this parameter is ok)

---

Removed.